addons/survey/models/survey_user.py

```diff
@@ -55,7 +55,7 @@ class SurveyUserInput(models.Model):
         ('unique_token', 'UNIQUE (access_token)', 'An access token must be unique!'),
     ]
 
-    @api.depends('user_input_line_ids.answer_score', 'user_input_line_ids.question_id')
+    @api.depends('user_input_line_ids.answer_score', 'user_input_line_ids.question_id', 'predefined_question_ids')
```

Comment:
Yep! In local, scoring is well computed without 'predefined_question_ids' dependance.
In runbot, predefined_question_ids is not yet recomputed before the scoring is computed, so that the scoring is wrong if you didn't answer every question of the survey.